src/topology/algebra/ordered.lean

```diff
@@ -1944,3 +1944,70 @@ lemma tendsto_of_monotone {ι α : Type*} [preorder ι] [topological_space α]
   tendsto f at_top at_top ∨ (∃ l, tendsto f at_top (𝓝 l)) :=
 if H : bdd_above (range f) then or.inr ⟨_, tendsto_at_top_csupr h_mono H⟩
 else or.inl $ tendsto_at_top_at_top_of_monotone' h_mono H
+
+lemma tendsto_neg_nhds {α : Type*} [decidable_linear_ordered_add_comm_group α]
+  [topological_space α] [topological_add_group α] {a : α} :
+  tendsto has_neg.neg (nhds a) (nhds (-a)) :=
```

Comment:
This lemma belongs to `topology/algebra/group`, and it doesn't need the `[decidable_linear_ordered_add_comm_group α]`: it just follows from the definition of `topological_add_group`.